src/configure/resources/messages.ts

```diff
@@ -113,6 +113,7 @@ export class Messages {
     public static AdoDifferentTenantError = " One potential reason can be the AAD tenant of your Azure DevOps repository is different than the one you have been logged-in to VSCode. Open 'Command Pallete' and select 'Azure: Sign Out' and then Sign-In to other tenant by invoking 'Deploy to Azure' extension";
     public static ConfiguringPipelineFailed = "Configuring provisioning pipeline failed due to %s";
```

Comment:
Change this message as well to Configuring github workflow failed due to %s

Reply:
Added